cmd/juju/application/utils/utils_test.go

```diff
@@ -187,7 +187,7 @@ func (s *utilsResourceSuite) TestGetUpgradeResourcesNotOriginUpload(c *gc.C) {
 
 func (s *utilsResourceSuite) TestGetUpgradeResourcesOriginUpload(c *gc.C) {
 	defer s.setupMocks(c).Finish()
-	s.charmClient.EXPECT().ListCharmResources(gomock.AssignableToTypeOf(&charm.URL{}), gomock.AssignableToTypeOf(apicharm.Origin{})).Return(nil, nil)
+	s.charmClient.EXPECT().ListCharmResources(gomock.AssignableToTypeOf(""), gomock.AssignableToTypeOf(apicharm.Origin{})).Return(nil, nil)
```

Comment:
Is it still needed to check if it is assignable to a string (also in cases above)?

Reply:
I guess not really.

Looking it at, I don't think these checks actually did anything, since ListCharmResources has always taken concrete ypes as it's params, if this check fails, it'll lead to a runtime error weather or not we check here